webdriver/tests/element_click/element_click.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click
+def test_scroll_into_view(session):
```

Comment:
Mind moving any scroll into view related tests to a file like `scroll_into_view.py` similar to what we have for `element_send_keys`?